### DISCUSSION TITLE
ADB over WiFi

### DISCUSSION THREAD

**Discussion by upintheairsheep** [yume-chan/ya-webadb#582] [Ideas]
ADB over WiFi is not present, and the ability to connect over wifi is not there either.

**Comment by yume-chan** [comment]
Toggle ADB over WiFi is added in the advanced page. Connect to devices using WebSockify won't be supported.

Still waiting for Direct Sockets API (https://github.com/WICG/direct-sockets), which adds native TCP and UDP sockets to Web apps.
